lexos/managers/lexos_file.py

```diff
@@ -128,7 +128,8 @@ def check_for_gutenberg(file_contents: str) -> bool:
         :return: a boolean representing if file is from Project Gutenberg.
         """
 
-        if re.search('Project Gutenberg', file_contents):
+        if re.search("\*\*\* START OF THIS PROJECT GUTENBERG.*?\*\*\*",
+                     file_contents):
```

Comment:
should the message be stored in a constant?

Reply:
kind of a hack, not sure how prevalent this "START OF ..." message is @gutenberg ?

Reply:
I agree this is a hack. We should use XML.

Reply:
@chantisnake, I'm not sure how we would use XML for this. Can you explain?

Reply:
Hum, I am guessing that they have a specific XML tag at a specific place?

I can be wrong.

Reply:
Sadly, they do not. The text is either plain text or HTML, but the HTML is not even that helpful.

Reply:
too bad, I need more investigation to give a definite answer to this problem

Reply:
they don't have any tags, and this is the only part of the Gutenberg boilerplates that is the same for all the Gutenberg files